src/main/java/org/alliancegenome/curation_api/model/bridges/IntegerValueBridge.java

```diff
@@ -0,0 +1,17 @@
+package org.alliancegenome.curation_api.model.bridges;
+
+import org.hibernate.search.mapper.pojo.bridge.ValueBridge;
+import org.hibernate.search.mapper.pojo.bridge.runtime.ValueBridgeToIndexedValueContext;
+
+public class IntegerValueBridge implements ValueBridge<Integer, String> {
+	@Override
+	public String toIndexedValue(Integer value, ValueBridgeToIndexedValueContext context) {
+		String padding = "0000000000000";
+		if (value == null) {
+			return padding;
+		}
+		String valueString = value.toString();
+		
+		return padding.substring(valueString.length()) + valueString;
+	}
+}
```

Comment:
@markquintontulloch here is another option.

return String.format("%013d", value.toString());